ui/src/components/entity/EntityAnnotationLayer.tsx

```diff
@@ -198,6 +197,13 @@ class EntityAnnotationLayer extends React.Component<Props, {}> {
             return null;
           }
 
+          /*
+           * Do not show abnormally large entities.
+           */
+          if (entity.attributes.tags.indexOf("large") !== -1) {
+            return null;
```

Comment:
though I will also walk it back in that it might be useful to do something for debugging if/when we hit this situation.

Reply:
I think you and Alex make a good point about this. I agree that it would be useful to be able to visualize large bounding boxes in the user interface, without having to make changes in the API. So let us start out with this code in the user interface, and we will migrate it to the API when we are feeling that we have done all that we want to do for filtering out bounding boxes.